endpoints/auction.go

```diff
@@ -199,56 +199,56 @@ func (a *auction) auction(w http.ResponseWriter, r *http.Request, _ httprouter.P
 				}
 			}
 			sentBids++
-			bidderRunner := recoverSafely(func(bidder *pbs.PBSBidder, blables pbsmetrics.AdapterLabels) {
+			bidderRunner := a.recoverSafely(func(bidder *pbs.PBSBidder, adapterlabels pbsmetrics.AdapterLabels) {
```

Comment:
Should be `adapterLabels`, as the go standard is to use camelcase

Reply:
This discussion should really be in https://github.com/prebid/prebid-server/pull/832, as these changes of _blables_ and _blabels_ to _adapterlabels_  originate there.  You're right that _adapterLables_ is preferable to _adapterlabels_.  Further investigation into variable naming reveals that maybe both names are too long (see https://talks.golang.org/2014/names.slide#6 and the subsequent two slides) and that maybe _alabels_ (or _aLabels_ ?) should be used.  Do you agree?

Reply:
I've updated the variable name to _alabels_ per further discussion with @hhhjort